src/indexer/indexAllowlistSingleClaimMinted.ts

```diff
@@ -0,0 +1,107 @@
+import { parseLeafClaimedEvent } from "@/parsing";
+import { getDeployment } from "@/utils";
+import { IndexerConfig, LeafClaimed } from "@/types/types";
+import { getContractEventsForChain } from "@/storage/getContractEventsForChain";
+import { updateLastBlockIndexedContractEvents } from "@/storage/updateLastBlockIndexedContractEvents";
+import { getLogsForContractEvents } from "@/monitoring/hypercerts";
+import { updateAllowlistRecordClaimed } from "@/storage/updateAllowlistRecordClaimed";
+
+/*
+ * This function indexes the logs of the ClaimStored event emitted by the HypercertMinter contract. Based on the last
```

Comment:
Docs for wrong event. I need to backtrack for the one I created but let's not create more debt